### ISSUE TITLE
Upcoming breaking change, channel beta: nullOk parameter was removed from Localizations.localeOf

### ISSUE THREAD

**Issue by paultz54** [marcglasberg/assorted_layout_widgets#9]
remove that and the example works

**Comment by marcglasberg** [comment]
Sorry, what is the problem you are having? Could you be more specific? Thanks.

**Comment by hampsterx** [comment]
```
assorted_layout_widgets-3.0.1/lib/src/text_one_line.dart:180:57: Error: No named parameter with the name 'nullOk'.
      locale: locale ?? Localizations.localeOf(context, nullOk: true),
```

```
Flutter (Channel beta, 1.26.0-17.4.pre, on Linux, locale en_NZ.UTF-8
```

Seems issue with beta/dev/master as the nullOk parameter was removed from `Localizations.localeOf`

**Comment by marcglasberg** [comment]
Fixed in version [4.0.0].
